### PR TITLE
sounds

### PR DESCRIPTION
**Pull request by BadRedSL**
sounds were changed + main octave was added + some fixes
@DumDereDum review pls

### CODE REVIEW

File: src/modules/piano.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def draw(self, img):
         return img
 
     def key_generator(self, spath, octave, key_num):
-        if (key_num > 14) or (octave != 4):
+        if (key_num > 14) or (octave not in [3, 4]):
```
Comment (DumDereDum) [inline src/modules/piano.py]: You have 3 octaves - 3, 4, 5
In that case, you need to do:
* `key_num > 21`
* `octave not in [3, 4, 5]`

Comment (BadRedSL) [reply src/modules/piano.py]: if we use 21 key_num and octave 4 or 5, we go beyond the border

Comment (BadRedSL) [reply src/modules/piano.py]: so either we write a very large condition on the keys, or we leave some restriction at this stage

Comment (DumDereDum) [reply src/modules/piano.py]: why? I have just tried this and it works. Try it on your machine

Comment (BadRedSL) [reply src/modules/piano.py]: this draws but crashes when trying to press a key

Comment (BadRedSL) [reply src/modules/piano.py]: because we don't have a sixth octave

**Pull request merged**